docs/articles/deployment/container.md

```diff
@@ -1,25 +1,164 @@
-It is very simple to run a Bytewax dataflow program in a container. In this section we will describe how to do that.
+In this chapter we are going to build a Docker image where we can run a dataflow, and present the prebuilt images we offer.
+
+## Your dataflow
+Let's start with a simple dataflow.
+
+Create a new file `dataflow.py` with the following content:
+
+```python
+from bytewax import operators as op
+from bytewax.connectors.stdio import StdOutSink
+from bytewax.dataflow import Dataflow
+from bytewax.testing import TestingSource
+
+flow = Dataflow("test")
+inp = op.input("in", flow, TestingSource(list(range(5))))
+op.output("out", inp, StdOutSink())
+```
+
+Run it locally to check everything's working fine:
+```
+$ python -m bytewax.run dataflow
+0
+1
+2
+3
+4
+```
+
+## Docker image
+You can run the dataflow inside a Docker image.
+You'll need an image with python support, and can just install bytewax and run the dataflow.
+
+Create a `Dockerfile` with the following content:
+```Dockerfile
+# Start from a debian slim with python support
+FROM python:3.11-slim-bullseye
+# Setup a workdir where we can put our dataflow
+WORKDIR /bytewax
+# Install bytewax and the dependencies you need here
+RUN pip install bytewax
+# Copy the dataflow in the workdir
+COPY dataflow.py dataflow.py
+# And run it
+# Note: use `-u` too to let python run unbuffered,
```

Comment:
Alternatively, you can use full cli arg and then we can skip this note

Reply:
I'm going to explicitely set the env var, so the setting doesn't depend on the parameters passed to `python`. What to you mean with "use the full cli args"?

Reply:
Sorry, my bad, I was 100% sure that there's a full version of that flag like `--unbuffered-stdin-and-stdiout`
+1 for env var